test/system/280-update.bats

```diff
@@ -127,4 +127,26 @@ device-write-iops   = /dev/zero:4000 | - | -
     fi
 }
 
+@test "podman update - set restart policy" {
+    touch ${PODMAN_TMPDIR}/sentinel
+    run_podman run --security-opt label=disable --name testctr -v ${PODMAN_TMPDIR}:/testdir -d $IMAGE sh -c "touch /testdir/alive; while test -e /testdir/sentinel; do sleep 0.1; done;"
```

Comment:
Missing `:Z`. Here's a shorter cleaner easier-to-find-problems version
```bash
@test "podman update - set restart policy" {
    local tmpdir=$PODMAN_TMPDIR
    touch ${tmpdir}/sentinel
    run_podman run --name testctr -v ${tmpdir}:/testdir:Z -d $IMAGE sh -c "touch /testdir/alive; while test -e /testdir/sentinel; do sleep 0.1; done;"

    # Ensure the container is alive
    wait_for_file ${tmpdir}/alive

    run_podman container inspect testctr --format "{{ .HostConfig.RestartPolicy.Name }}"
    is "$output" "no"

    run_podman update --restart always testctr

    run_podman container inspect testctr --format "{{ .HostConfig.RestartPolicy.Name }}"
    is "$output" "always"

    rm -f ${tmpdir}/alive ${tmpdir}/sentinel

    # Restart should ensure that the container comes back up and recreates the file
    wait_for_file ${tmpdir}/alive
}
```

Reply:
It was the *first* `wait_for_file` that was barfing. I moved it closer to the source, to make it more obvious.

Reply:
Um I think you forgot the `:Z`

Reply:
Just using `--security-opt label=disable` instead so we don't relabel the tmpdir.